### PR TITLE
[New] Client credentials for Timescale Cloud

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Documented how to create client credentials for programmatic access to Cloud
Moved MFA under security

# Links

Fixes https://github.com/timescale/docs-private/issues/82

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: cloud/security/client-credentials.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+access to Timescale resources.
+
+<Procedure>
+
+### Creating client credentails
+
+1.  [Log in to your Timescale Cloud account][cloud-login].
+1.  Navigate to the `Project Settings` page to create client credentials for
+    your project.
+1.  In the `Project Settings` page, click `Create credentails`.
```
Comment (lukas-bernert) [inline cloud/security/client-credentials.md:31]:
```suggestion
1.  In the `Project Settings` page, click `Create credentials`.
```

File: cloud/security/client-credentials.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+When you create client credentails, a public key and a private key is generated.
+These keys act as the username and password for a programmatic client
+applications. It is important that you save these keys in a safe place. You can
+also delete these client credentials when the client applications no longer need
+access to Timescale resources.
+
+<Procedure>
+
+### Creating client credentails
```
Comment (lukas-bernert) [inline cloud/security/client-credentials.md:26]:
```suggestion
### Creating client credentials
```

File: cloud/security/client-credentials.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# Client credentials
+
+The client credentials allows you to programmatically access resources instead
+of using your username and password. You can generate multiple client
+credentials for different applicationss or use cases rather than a single set of
+user credentials for everything.
+
+## Creating client credentials
+
+When you create client credentails, a public key and a private key is generated.
```
Comment (lukas-bernert) [inline cloud/security/client-credentials.md:18]:
```suggestion
When you create client credentials, a public key and a private key is generated.
```

**Review by lukas-bernert — APPROVED** [review]

**Comment by Loquacity** [comment]
This might get all sorts of merge conflicts after the reorg goes live. Just leaving this here as a note, I'm happy to swoop back in and fix things later on if needed 🤞

**Comment by Loquacity** [comment]
> This might get all sorts of merge conflicts after the reorg goes live. Just leaving this here as a note, I'm happy to swoop back in and fix things later on if needed 🤞

Should be all good now, let me know if anything seems a bit weird 👍

File: cloud/page-index/page-index.js [hunk 4]
```diff
@@ -195,6 +190,11 @@ module.exports = [
             excerpt:
               "Connect to Timescale Cloud with a stricter SSL mode of verify-ca or verify-full",
           },
+          {
+            title: "Multi-factor Authentication",
```
Comment (Loquacity) [inline cloud/page-index/page-index.js:194]:
```suggestion
            title: "Multi-factor authentication",
```

File: cloud/security/client-credentials.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+excerpt: Client credentials to programmatically access your Timescale Cloud account
+product: cloud
+keywords: [client credentials, accounts, prgrammatic access]
+tags: [authentication tokens]
+---
+
+# Client credentials
+
+The client credentials allows you to programmatically access resources instead
+of using your username and password. You can generate multiple client
```
Comment (Loquacity) [inline cloud/security/client-credentials.md:12]:
```suggestion
You can use client credentials to programmatically access resources instead
of using your username and password. You can generate multiple client
```

File: cloud/security/client-credentials.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+credentials for different applicationss or use cases rather than a single set of
+user credentials for everything.
+
+## Creating client credentials
+
+When you create client credentials, a public key and a private key is generated.
+These keys act as the username and password for a programmatic client
```
Comment (Loquacity) [inline cloud/security/client-credentials.md:19]:
```suggestion
These keys act as the username and password for programmatic client
```

File: cloud/security/client-credentials.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+### Deleting client credentials
+
+<Procedure>
+
+1.  [Log in to your Timescale Cloud account][cloud-login].
+1.  Navigate to the `Project Settings` page to view client credentials for
+    your project.
+1.  In the `Project Settings` page, click the ⋮ menu of the client credential,
+    and select `Delete`.
+1.  In the `Are you sure` dialog, type the name of the client credentail, and
```
Comment (Loquacity) [inline cloud/security/client-credentials.md:54]:
```suggestion
1.  In the `Are you sure` dialog, type the name of the client credential, and
```

File: cloud/security/client-credentials.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+product: cloud
+keywords: [client credentials, accounts, prgrammatic access]
+tags: [authentication tokens]
+---
+
+# Client credentials
+
+You can use client credentials to programmatically access resources instead
+of using your username and password. You can generate multiple client
+credentials for different applicationss or use cases rather than a single set of
```
Comment (Loquacity) [inline cloud/security/client-credentials.md:13]:
```suggestion
credentials for different applications or use cases rather than a single set of
```

**Review by Loquacity — APPROVED** [review]
Just a typo

**Pull request merged**